website/api/zkvm/zkvm_overview.md

```diff
@@ -65,6 +65,15 @@ Once you've built a zkVM application, you can generate proofs on your own machin
 For local proof generation, we recommend a machine with at least 16GB of RAM.
 Options for generating proofs using GPU acceleration and skipping proof generation are documented in the [feature flags].
 
+## Key Features
+
+There are several key features that distinguish RISC Zero's zkVM from others. These features enable developers to more easily develop zk powerful applications.
```

Comment:
It feels a bit awkward to allude to "several key features" and then only list one. I would either drop this sentence, or add another subsection, maybe for instance this bit adapted from our website?
> Micro Architecture
>
> The zkVM is a verifiable computer that works like a real embedded RISC-V micro-processor. Write ZK proofs just like any other code.

Reply:
good point... I was thinking that this would be the first heading of a few points but it does read a little awkward at this point..

Reply:
yeah it makes sense as soon as we flesh this out with more points, but until that happens its weird. You could always drop for now and grab the language out of this PR again once we have more points ready.